ui/src/actions/literature.ts

```diff
@@ -158,6 +185,29 @@ export function fetchLiteratureReferences(
   };
 }
 
+export function fetchReferencesDiff(  
+  recordId: number,
+  prevId: number,
```

Comment:
maybe you can name those vars like `prevVersionNumber` or `prevRevisionId` so that it won't be confusing with any kind of record ID